lib/blueprinter/errors.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
```

Comment:
The new autoload functionality is new to me, so this is likely a basic question - why is this autoload separate from the others in `lib/blueprinter.rb`?

Is it more of an organization thing? Like to avoid putting all of the required autoloads in one massive file? 

Or is there some valuable functional difference by putting it here? Do we gain anything performance-wise by separating this out?

Just trying to further my understanding as I'm learning about it :)

Reply:
Yeah, purely organizational.